### DISCUSSION TITLE
cpu at 100%, dns flooded, bookmarks not visible in nextcloud

### DISCUSSION THREAD

**Discussion by m4rtini2** [floccusaddon/floccus#1144] [Q&A] [answered]
Hello,
i just installed this app and the browser extension and have some issues with it.
While syncing the bookmarks nextcloud's cpu usage goes up to 100% and it becomes very unresponsive. Also the dns requests created by the bookmark app brake my pi-hole stats as the high spikes makes the other request nearly invisible. Like 4000 request from bookmarks and 30 from the rest of my devices.
Please make the bookmark checks optional and the synchronisation less cpu intensive.

Nextcloud is running in a virtual machine so the high cpu load is no problem for the server but incovinient for the user. 
The vm does not have own storage so the bookmark folder is a subfolder of my documents folder that is linked to nextcloud via external storage for the user i want to sync the bookmarks. Like: /server/user/documents/bookmarks with all rights to create, edit, delete the files on it. This is also the path i entered in floccus as "server directory". Unfortunately on nextcloud this folder stays empty. Floccus puts the bookmarks somewhere else but i can not find this folder. But i need the bookmarks to be in this folder.
How to tell floccus to save the bookmarks in a folder on external storage?

nextcloud  v23.0.4
bookmarks 10.3.1
external storage connected to nextcloud via smb/cifs
floccus 4.13.1
pi-hole message "Long-term load (15min avg) larger than number of processors: 1.2 > 1
This may slow down DNS resolution and can cause bottlenecks."

**Comment by marcelklehr** [comment, marked as answer]
> But i need the bookmarks to be in this folder. How to tell floccus to save the bookmarks in a folder on external storage?

You can either sync to Nextcloud Bookmarks, the bookmarks will then be in the bookmarks app, or you can sync to nextcloud via webdav, the bookmarks will then be in an xbel file in your nextcloud files, which you can put on external storage.
